moped-editor/src/layouts/DashboardLayout/NavBar/MobileDropdownMenu.js

```diff
@@ -65,7 +65,7 @@ const MobileDropdownMenu = () => {
             key={item.href}
             onClick={() => {
               handleMobileClose();
-              navigate(item.link);
+              navigate(item.href);
```

Comment:
anyone notice that the mobile menu navigation links to dashboard/project/staff previously didn't work??? 🙈

Reply:
😳

Reply:
yowza - this had me confused for a second but I was still clicking external links in staging instead of the router ones! 🤦